typescript/sdk/src/token/schemas.ts

```diff
@@ -17,11 +17,12 @@ export const TokenMetadataSchema = z.object({
 export const CollateralConfigSchema = TokenMetadataSchema.partial().extend({
   type: z.enum([
     TokenType.collateral,
-    TokenType.collateralXERC20,
+    TokenType.collateralVault,
+    TokenType.XERC20,
+    TokenType.XERC20Lockbox,
     TokenType.collateralFiat,
-    TokenType.collateralUri,
     TokenType.fastCollateral,
-    TokenType.collateralVault,
+    TokenType.collateralUri,
```

Comment:
nit: small taste thing but since we chose `XERC20` above and not `XErc20`, I feel like `collateralURI` is more appropriate

Reply:
this existed before my PR